backend/services/progress_tracker.py

```
@@ -0,0 +1,96 @@
+from enum import Enum
+import asyncio
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)